### DISCUSSION TITLE
New sync command. where to put on unraid?

### DISCUSSION THREAD

**Discussion by gitttt54** [Taxel/PlexTraktSync#1320] [Support] [answered]
Hello,

first, I have long time missed those nice (new for me) commands, like "sync", "plex-login", "trakt-login" and others. They are very useful and make life easier. Thank you!

I'm on unraid and have added the "sync" command to my cron. works good.

Only, when I start the PTS container manually, I get the "depreciated" message, which I'd like to fix by adding the command "sync" to my docker containers configuration.

I have read the docs here to find out, where to put that sync command, but could not find any info I could understand or use.
(btw: the docs are very detailed and helpful, but I guess, I'm too dumb)

I have no clue of "docker compose" or where to put that "sync" command in my config here:
<img width="1056" alt="Bildschirmfoto 2022-12-25 um 10 36 30" src="https://user-images.githubusercontent.com/96403800/209463149-52b8a04b-2d58-4fb4-a68b-35977a62adc9.png">

Do I have to add a new variable ? like this?

<img width="808" alt="Bildschirmfoto 2022-12-25 um 10 38 02" src="https://user-images.githubusercontent.com/96403800/209463184-1c1349a2-69c1-4921-adf5-47e94db53379.png">

Would someone please help?

Thank you ;-)

**Comment by simonc56** [comment, marked as answer]
From [reddit ](https://www.reddit.com/r/unRAID/comments/t602dl/help_with_plex_trakt_sync/):

Edit the container, in the top right, click the toggle at basic view onto advanced view, fill your argument into the "post arguments" box

**Reply by gitttt54** [reply]
Thank you! That seems to do the job!

<img width="1248" alt="Bildschirmfoto 2022-12-30 um 15 36 49" src="https://user-images.githubusercontent.com/96403800/210082533-1932a32f-ef16-48ab-af8f-eca9d96abcc5.png"><img width="1245" alt="Bildschirmfoto 2022-12-30 um 15 37 16" src="https://user-images.githubusercontent.com/96403800/210082434-bf8e6029-c5dc-41a6-b922-2e702dfa7193.png">
